code/modules/mining/mine_turfs.dm

```diff
@@ -605,6 +605,10 @@
 			return 0
 	if(ishuman(AM))
 		var/mob/living/carbon/human/H = AM
+		if(istype(H.shoes, /obj/item/clothing/shoes/bhop))
```

Comment:
shouldn't be necessary, the boots throw you over the chasm, and Joan recently made thrown things ignore chasm falling.